frontend/packages/integration-tests-cypress/tests/crud/namespace-crud.spec.ts

```diff
@@ -53,54 +56,113 @@ describe('Namespace', () => {
     );
-    cy.get(`[data-test-rows="resource-row"]`)
-      .first()
-      .find('a')
-      .first()
+    listPage.rows.clickFirstLinkInFirstRow();
+    detailsPage.isLoaded();
+    projectDropdown.shouldNotContain(AllProjects); // after drilldown to Details page, project should be specific to resource
+    detailsPage
+      .breadcrumb(0)
+      .contains('Secrets')
       .click();
-    projectDropdown.shouldNotContain('All Projects'); // after drilldown to Details page, project should be specific to resource
-    cy.byLegacyTestID('breadcrumb-link-0').click();
-    projectDropdown.shouldContain('All Projects');
+    listPage.rows.shouldBeLoaded();
+    projectDropdown.shouldContain(AllProjects);
   });
 
   it('nav and breadcrumbs restores last selected Project when navigating from details to list view', () => {
     nav.sidenav.clickNavLink(['Workloads', 'Secrets']);
-    projectDropdown.selectProject('default');
-    projectDropdown.shouldContain('default');
-    cy.get(`[data-test-rows="resource-row"]`)
-      .first()
-      .find('a')
-      .first()
-      .click();
-    projectDropdown.shouldContain('default');
+    projectDropdown.selectProject(defaultProjectName);
+    projectDropdown.shouldContain(defaultProjectName);
+    listPage.rows.clickFirstLinkInFirstRow();
+    detailsPage.isLoaded();
+    projectDropdown.shouldContain(defaultProjectName);
     nav.sidenav.clickNavLink(['Workloads', 'Secrets']);
-    projectDropdown.shouldContain('default');
-    cy.get(`[data-test-rows="resource-row"]`)
-      .first()
-      .find('a')
-      .first()
+    projectDropdown.shouldContain(defaultProjectName);
+    listPage.rows.clickFirstLinkInFirstRow();
+    detailsPage.isLoaded();
+    projectDropdown.shouldContain(defaultProjectName);
+    detailsPage
+      .breadcrumb(0)
+      .contains('Secrets')
       .click();
-    projectDropdown.shouldContain('default');
-    cy.byLegacyTestID('breadcrumb-link-0').click();
-    projectDropdown.shouldContain('default');
+    listPage.rows.shouldBeLoaded();
+    projectDropdown.shouldContain(defaultProjectName);
+  });
+
+  ['Roles', 'RoleBindings'].forEach((rolesOrBindings) => {
+    it(`tests ${rolesOrBindings} and Cluster${rolesOrBindings} detail page breadcrumbs`, () => {
+      nav.sidenav.clickNavLink(['User Management', rolesOrBindings]);
+
+      cy.log(`test ${rolesOrBindings} & All Projects`);
+      projectDropdown.selectProject(AllProjects);
+      listPage.filter.by('cluster');
+      listPage.filter.numberOfActiveFiltersShouldBe(1);
+      listPage.rows.clickFirstLinkInFirstRow();
+      detailsPage.isLoaded();
+      projectDropdown.shouldNotExist();
+      detailsPage
+        .breadcrumb(0)
+        .contains(rolesOrBindings)
+        .click();
+      listPage.rows.shouldBeLoaded();
+      projectDropdown.shouldContain(AllProjects);
+      listPage.titleShouldHaveText(rolesOrBindings);
+
+      cy.log(`test Cluster${rolesOrBindings} & openshift-console project`);
```

Comment:
Looks like the log messages are out of date with the code now that we changed the namespace.

Reply:
Hi, yes, caught those as well :-) Should be pushing new code soon.

Reply:
.. done.